.kitchen.yml

```diff
@@ -41,4 +41,11 @@ suites:
     attributes:
     verifier:
       inspec_tests:
-        - https://github.com/dev-sec/windows-hardening-benchmark
\ No newline at end of file
+        - https://github.com/dev-sec/windows-hardening-benchmark
+  
+  - name: CIS_2012r2_L1
```

Comment:
I feel like we should merge this and add the tests to https://github.com/dev-sec/windows-baseline